### PR TITLE
Tablet compatibility

### PR DESCRIPTION
**Pull request by elucidium**
### Summary of changes
- `$DESKTOP_WIDTH` is now 1240px and a new constant `$TABLET_WIDTH` is 672px. The range in between is detected as a tablet screen, and landscape/portrait orientation further conditioned upon in `styles/main.scss`
- Desktop elements were selectively resized to look better at tablet resolutions, primarily done by changing hard-coded heights and font-sizes to be in units of `%` and `vh` respectively (c.f. `styles/tablet/index.css`)
- The `NavItem` circles now react to the `touchstart` event in addition to `mouseenter` in order to allow the landscape tablet mode to be navigable (note: not completely sure if this interferes with the FSM model, but I don't see any reason why it would since that only deals with the mobile view)
- Set starting height of the `#mobile-menu-line-list` to `-100%` instead of the previous hardcoded value because it would show in portrait orientation

### Outstanding notes/issues
- For portrait orientation, I've left the sizes as they are in the mobile code since the relative sizes of everything seem to be a bit up in the air based upon discourse in the Figma—this can all be changed to scale with screen size once everything is finalized (@sabrinazhai)
- Related note: the breakpoint for when "The Marianas Trench" currently bleeds into two lines is ~489px, which means that basically all mobile screens currently see the following. Either font sizes need to be adjusted, we need to change `$LINE_ELEMENT_HEIGHT`, or the height needs to be responsive (@mikinty I think? Also depends on designer feedback)
Screen Shot 2021-01-01 at 14 21 10

#### Landscape (largest)
Screen Shot 2021-01-01 at 16 01 35

#### Landscape (smallest)
Screen Shot 2021-01-01 at 16 02 01

#### Portrait (iPad Pro 12.9")
Screen Shot 2021-01-01 at 16 09 23

#### Portrait (iPad Mini)
Screen Shot 2021-01-01 at 16 08 49

### CODE REVIEW

File: src/styles/tablet/index.scss
Comment (mikinty) [inline src/styles/tablet/index.scss]: Do you need this important? I am very wary of using important unless it's _absolutely_ necessary

**Review by mikinty — CHANGES_REQUESTED** [review]

**Comment by mikinty** [comment]
Can you also fix merge conflicts? Wait until desktop is landed though since I think that one is pretty much ready

File: src/styles/constants.scss
Comment (mikinty) [inline src/styles/constants.scss:10]: Let's test on iPad (and maybe surface since i happen to have one) and see if this works

File: src/styles/main.scss
Comment (mikinty) [inline src/styles/main.scss:36]: nice queries

File: src/styles/tablet/index.scss
Comment (mikinty) [inline src/styles/tablet/index.scss]: extra space

File: src/styles/tablet/index.scss
Comment (elucidium) [reply src/styles/tablet/index.scss]: resolved by refactoring

**Comment by elucidium** [comment]
re: merge conflicts, just let me know when to take a look

**Comment by mikinty** [comment]
> re: merge conflicts, just let me know when to take a look

Should be done now you can merge

**Review by mikinty — APPROVED** [review]

**Pull request merged**
